src/stores/utils/gas.js

```diff
@@ -0,0 +1,27 @@
+const { toWei } = require('web3').utils
+
+export async function fetchGasPrice({ bridgeContract, oracleFn }) {
+  let gasPrice = null
+  try {
+    gasPrice = await oracleFn()
+  } catch (e) {
+    console.error(`Gas Price API is not available. ${e.message}`)
```

Comment:
This will appear even if the corresponding environment variable declaring the gas price oracle url is not defined. Please consider to not call gas price oracle at all in such case to not flood the logs.

Reply:
Done